src/vl-http-506-message.js

```diff
@@ -0,0 +1,14 @@
+import {define} from '/node_modules/vl-ui-core/dist/vl-core.js';
+import {VlHttpErrorMessage} from '/src/vl-http-error-message.js';
+
+define('vl-http-506-message', class extends VlHttpErrorMessage {
+  constructor() {
+    super({
+      title: 'Sorry, oneindige cirkels',
+      image: 'https://cdn.milieuinfo.be/http-error-message-assets/LATEST/img/unexpected-error.svg',
+      imageAlt: 'Oneindige cirkels',
+      text: `<p>Uw verzoek veroorzaakt intern oneindige cirkels.</p>`,
```

Comment:
gaat de gebruiker dit verstaan deze 'cirkels'? Slaat dit op 'infinite loop'? Ik zou zelf niet meteen weten wat hier een beter oplossing voor is, "Oneindige lussen" klinkt ook niet veel beter.

Reply:
Misschien is 'interactie' beter hier. Ik heb nu maar zo goed als mogelijk de teksten toegevoegd. Deze zullen vermoedelijk nog in samenspraak met infra verfijnd worden.